tests/playwright/sanity/modules/floating-buttons/floating-elements-admin.test.ts

```diff
@@ -0,0 +1,182 @@
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		await floatingElPage.deleteAllFloatingButtons();
+		await page.close();
+	} );
+
+	test( 'Verify editor behavior by creating post through API and also FE behavior', async (
+		{
+			browser,
+			page,
+			apiRequests,
+		}, testInfo ) => {
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		const editor = await floatingElPage.createFloatingButtonWithAPI();
+		await floatingElPage.waitForPanel();
+		const navigatorId = 'header [aria-label="Structure"]';
+		const navigator = page.locator( navigatorId );
+		// We need to check that the normal elements are not present, that the navigator is visible and the title is correct.
+		await expect( navigator ).toBeHidden();
+		const addButton = page.locator( 'header [aria-label="Add Element"]' );
+		await expect( addButton ).toBeHidden();
+		const panelTitle = page.locator( '#elementor-panel-header-title' );
+		await expect( panelTitle ).toHaveText( 'Edit Single Chat' );
+
+		// Check that we are seeing our custom advanced tab and that the normal advanced tab is hidden.
+		const navigationContainer = page.locator( '.elementor-panel-navigation' );
+		const buttons = navigationContainer.locator( 'button' );
+		const visibleButtons = buttons.filter( { has: page.locator( ':visible' ) } );
+		await expect( visibleButtons ).toHaveCount( 3 );
+		const advancedSettingsCustom = page.locator( '.elementor-tab-control-advanced-tab-floating-buttons' );
+		await expect( advancedSettingsCustom ).toBeVisible();
+		await advancedSettingsCustom.click();
+
+		const elementorControlsContainer = page.locator( '#elementor-controls' );
+		const controlsToCheck = [
+			'.elementor-control-advanced_layout_section',
+			'.elementor-control-advanced_responsive_section',
+			'.elementor-control-advanced_custom_controls_section',
+			'.elementor-control-section_custom_css_pro',
+			'.elementor-control-section_custom_attributes_pro',
+		];
+
+		controlsToCheck.forEach( async ( control ) => {
+			const element = elementorControlsContainer.locator( control );
+			await expect( element ).toBeVisible();
+		} );
+
+		const sections = elementorControlsContainer.locator( '.elementor-control-type-section' );
+		await expect( sections ).toHaveCount( 5 );
+
+		const addNewWidget = editor.getPreviewFrame().locator( EditorSelectors.addNewSection );
+		await expect( addNewWidget ).toBeHidden();
+		const container = editor.getPreviewFrame().locator( EditorSelectors.container );
+		await expect( container ).toBeVisible();
+		const contactButtonWidget = container.locator( EditorSelectors.getWidgetByName( 'contact-buttons' ) );
+		await expect( contactButtonWidget ).toBeVisible();
+		const contactButtonActionsContainer = container.locator( '.elementor-editor-container-settings' );
+		const actions = contactButtonActionsContainer.locator( 'li' );
```

Comment:
I don't think we need this assertion

Reply:
@louiswol94 I think we do because this tests that only the Delete Container button exists, we have code added for that part, to remove the other two buttons.

Reply:
Ah ok!